src/sass/_pattern_icon.scss

```diff
@@ -2,11 +2,83 @@
+}
+
+@mixin lxdui-icon-certificate() {
+  background-image: url("data:image/svg+xml,%3Csvg viewBox='0 0 16 16' width='16px' height='16px' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath%20fill-rule%3D%22evenodd%22%20clip-rule%3D%22evenodd%22%20d%3D%22M13%202C14.1046%202%2015%202.89543%2015%204V11C15%2012.1046%2014.1046%2013%2013%2013H12V11.5H13C13.2455%2011.5%2013.4496%2011.3231%2013.4919%2011.0899L13.5%2011V4C13.5%203.75454%2013.3231%203.55039%2013.0899%203.50806L13%203.5H3C2.75454%203.5%202.55039%203.67688%202.50806%203.91012L2.5%204V11C2.5%2011.2455%202.67688%2011.4496%202.91012%2011.4919L3%2011.5H7V13H3C1.89543%2013%201%2012.1046%201%2011V4C1%202.89543%201.89543%202%203%202H13ZM9.5%207C10.8807%207%2012%208.11929%2012%209.5C12%2010.3175%2011.6076%2011.0433%2011.001%2011.4995L11%2016L9.5%2014.9561L8%2016L8.00004%2011.5002C7.3928%2011.0441%207%2010.3179%207%209.5C7%208.11929%208.11929%207%209.5%207ZM8.5%209.5C8.5%208.94772%208.94772%208.5%209.5%208.5C10.0523%208.5%2010.5%208.94772%2010.5%209.5C10.5%2010.0523%2010.0523%2010.5%209.5%2010.5C8.94772%2010.5%208.5%2010.0523%208.5%209.5Z%22%20fill%3D%22black%22%2F%3E%3C%2Fsvg%3E");
+}
+
```

Comment:
We should update vanilla first, so we can import those instead of defining them here.

Reply:
still waiting for e2e to pass on #947